### PR TITLE
fix: update tabindex value(CXSPA-948) 

### PR DESCRIPTION
**Pull request by kimhw0630**
Closes: https://jira.tools.sap/browse/CXSPA-948

tabindex value will get update by input value

### CODE REVIEW

File: feature-libs/storefinder/components/store-finder-search/store-finder-search.component.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
         ></cx-icon>
         <cx-icon
           *cxFeatureLevel="'5.2'"
-          tabindex="-1"
+          [attr.tabindex]="(queryInput.value && queryInput.value.length) ? 0 : -1"
```
Comment (justinlee01) [inline feature-libs/storefinder/components/store-finder-search/store-finder-search.component.html]: I believe this is allowed in recent versions of Angular

```suggestion
          [attr.tabindex]="(queryInput.value?.length) ? 0 : -1"
```

**Review by justinlee01 — COMMENTED** [review]
Not a breaking change?

File: feature-libs/storefinder/components/store-finder-search/store-finder-search.component.html (hunk 1 above)
Comment (kimhw0630) [reply feature-libs/storefinder/components/store-finder-search/store-finder-search.component.html]: i will update the code and it already has feature level check

**Review by justinlee01 — APPROVED** [review]

**Review by developpeurweb — APPROVED** [review]
Works well with VO and JAWS

**Pull request merged**